_posts/2020-2-25-introduction-to-quantization-on-pytorch.md

```diff
@@ -0,0 +1,147 @@
+   ```# set quantization config for server (x86) deploymentmyModel.qconfig = torch.quantization.get_default_config('fbgemm') # insert observerstorch.quantization.prepare(myModel,inplace=True) # Calibrate the model and collect statistics# convert to quantized versiontorch.quantization.convert(myModel, inplace=True)```
+
+### **Quantization Aware Training**
+**Quantization-aware training(QAT)** is the third method, and the one that typically results in highest accuracy of these three. With QAT, all weights and activations are “fake quantized” during both the forward and backward passes of training: that is, float values are rounded to mimic int8 values, but all computations are still done with floating point numbers. Thus, all the weight adjustments during training are made while “aware” of the fact that the model will ultimately be quantized; after quantizing, therefore, this method usually yields higher accuracy than the other two methods.
+* **PyTorch API**:
+  * `torch.quantization.prepare_qat` inserts fake quantization modules to model quantization.
+  * Mimicking the static quantization API, `torch.quantization.convert` actually quantizes the model once training is complete.
+
+For example, in [the end-to-end example](https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html), we load in a pre-trained model asqat_model, then we simply perform quantization-aware training using:
+* ```# specify quantization config for QATqat_model.qconfig=torch.quantization.get_default_qat_qconfig('fbgemm')# prepare QATtorch.quantization.prepare_qat(qat_model,inplace=True)# convert to quantized version, removing dropout, to check for accuracy on each epochquantized_model=torch.quantization.convert(qat_model.eval(),inplace=False)```
+
+#### **Device and Operator Support**  
+Quantization support is restricted to a subset of available operators, depending on the method being used, for a list of supported operators, please see the documentation at https://pytorch.org/docs/stablquantization.html.
+
+The set of available operators and the quantization numerics also depend on the backend being used to run quantized models. Currently quantized operators are supported only for CPU inference in the following backends: x86 and ARM. Both the quantization configuration (how tensors should be quantized and the quantized kernels (arithmetic with quantized tensors) are backend dependent. One can specify the backend by doing: ```importtorchbackend='fbgemm'# 'fbgemm' for server, 'qnnpack' for mobilemy_modeqconfig =torch.quantization.get_default_qconfig(backend)# prepare and convert model# Set the backend owhich the quantized kernels need to be runtorch.backends.quantized.engine=backend```
```

Comment:
You're missing a space between `import` and `torch`. I also think the code block will look better if it starts on a new line.